style.css

```diff
@@ -0,0 +1,214 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: Georgia;
+	letter-spacing: 1px;
+	background-color: #e1e5ee;
+}
+
+header {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	padding: 15px 0;
+	background-color: #c7ccdb;
+	color: #2a324b;
+	margin: auto;
+}
+
+.banner {
+	padding: 5px;
+	width: 80%;
+	background-color: #c7ccdb;
+	color: #2a324b;
+	margin: 20px auto;
+	border-style: solid;
+	border-width: 1px;
+	border-color: #2a324b;
+	text-align: center;
+}
+.banner:hover {
+	letter-spacing: 3px;
+	transition-duration: 0.2s;
+}
+
+main {
+	background-image: url("images/floor.jpg");
+	background-position: center;
+	background-size: cover;
+	filter: grayscale(100%);
+	padding: 210px;
+	width: 60%;
+	margin: auto;
+	border: 1px solid #6d7275;
+	color: #c7ccdb;
+}
+
+.btn:link,
+.btn:visited {
+	text-decoration: none;
+	color: #e1e5ee;
+	border: 1px solid #e1e5ee;
+	padding: 4px;
+	border-radius: 4px;
+}
+
+.btn:hover {
+	background-color: #e1e5ee;
+	color: #6d7275;
+	transition-duration: 0.5s;
+	border-color: #6d7275;
+}
+
+
+.items {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
+	width: 80%;
+	justify-content: space-between;
+	margin: auto;
+	color: #2a324b;
+	gap: 10px;
+}
+
+
+.portfolio-item {
+	padding: 0 20px;
+	min-height: 200px;
+}
+
+h5 {
+	padding: 0 0 50px 0;
+
+}
+
```

Comment:
For elements .pi-image1, .pi-image2 etc, instead of the same attributes on each, use

.portfolio-item {
background-size: cover;
transition-duration: 0.5s;
}